samples/AppConfiguration/AppConfiguration/Generated/Operations/AllOperations.cs

```diff
@@ -61,6 +54,14 @@ internal HttpMessage CreateGetKeysRequest(string? name, string? after, string? a
             {
                 request.Uri.AppendQuery("After", after, true);
             }
+            if (syncToken != null)
```

Comment:
Why is this reordered?

Reply:
Because Uri appending was moved to where Uri Resetting is. Originally, it was in an if-statement since they are both related to each other. Now that I'm not using that if statement, I'm leaving it so that Uri Reset and Uri Appends are one after the other. Before, Header appending was in-between both of them.

Reply:
Ok, keeping thing next to each other makes sense.